lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsFileSystemConfig.java

```diff
@@ -120,6 +122,19 @@ public GcsFileSystemConfig setProjectId(String projectId)
         return this;
     }
 
+    public Optional<String> getEndpoint()
+    {
+        return endpoint;
+    }
+
+    @ConfigDescription("Endpoint to use for GCS requests")
+    @Config("gcs.endpoint")
+    public GcsFileSystemConfig setEndpoint(Optional<String> endpoint)
+    {
+        this.endpoint = endpoint;
+        return this;
+    }
+
```

Comment:
Move these methods above `getProjectId` for consistency with the field's order.

Reply:
I did this other way around. Moved it below projectId to match the actual order of methods